tools/src/test/java/com/nedap/archie/rmobjectvalidator/RmObjectValidatorTest.java

```diff
@@ -26,43 +31,55 @@
 import static org.junit.Assert.assertTrue;
 
 public class RmObjectValidatorTest {
-
     private ADLParser parser;
     private TestUtil testUtil;
+    InMemoryFullArchetypeRepository emptyRepo;
     private RMObjectValidator validator;
 
+
     @Before
     public void setup() {
         parser = new ADLParser();
         testUtil = new TestUtil();
-        validator = new RMObjectValidator(ArchieRMInfoLookup.getInstance());
+
+        emptyRepo = new InMemoryFullArchetypeRepository();;
+        validator = new RMObjectValidator(ArchieRMInfoLookup.getInstance(), (templateId) -> null);
```

Comment:
In this file you use `emptyRepo`, `(templateId) -> null` and `new InMemoryFullArchetypeRepository()` as second parameter `RMObjectValidator`. They all do the same, or do I miss something?

Reply:
You're right, that's a bit inconsistent. I'll make it consistent again.